src/components/map/MapService.js

```diff
@@ -851,7 +852,7 @@ goog.require('ga_urlutils_service');
         // Returns a list of WMS servers using a template
         // (e.g. //wms{s}.geo.admin.ch) and an array of
         // subdomains (e.g. ['', '1', ...]).
-        var getWmsUrls = function(tpl, subdomains) {
+        var getImageryUrls = function(tpl, subdomains) {
```

Comment:
May be change the description of the function, because now it's not only used for WMS